internal/colorize/cropped.go

```diff
@@ -0,0 +1,61 @@
+package colorize
+
+type Entry struct {
+	Line   string
+	Length int
+}
+
+func GetCroppedText(text string, maxLen int) []Entry {
+	entries := make([]Entry, 0)
+	entryText := ""
+	currentPosition := 0
+	runesWritten := 0
+	matches := colorRx.FindAllStringSubmatchIndex(text, -1)
+	runeText := []rune(text)
+	last := len(runeText) - 1
```

Comment:
```suggestion
func GetCroppedText(text []rune, lineSize int) []Entry {
        var string line // the line we're building, including the color codes
        var int plainTextPosition // plainText = minus color codes
        var int plainTextWritten
        
	entries := make([]Entry, 0)
	colorCodes := colorRx.FindAllStringSubmatchIndex(text, -1)
```

Trying to make your variables a bit more consistent and easier to digest.

This renames (make sure to update the code):

- maxLen > lineSize
- entryText > line
- currentPosition > plainTextPosition
- runesWritten > plainTextWritten
- matches > colorCodes
- last > dropped, just do `len(runeText) - 1` in your code

Open to other suggestions, but end goal is this code overall still needs to improve in readability. Variable names are perfect for telling the story of your code, so be mindful of how you name them.